packages/api/src/record/validators.ts

```diff
@@ -42,3 +42,28 @@ export function validatePatchRecordRequest(
     throw validation.error;
   }
 }
+
+export function validateCopyRecordRequest(
+  data: unknown
+): asserts data is CopyRecordRequest {
+  const validation = Joi.object()
+    .keys({
+      ...fieldsFromUserAuthentication,
+      RecordVO: {
```

Comment:
The concept of a VO isn't used in `stela`

Reply:
Ah, so we don't want to keep backwards compatibility then? I was trying to keep the request format the same.

Reply:
Yeah, this does not need to be backwards compatible